dotCMS/src/main/java/com/dotmarketing/business/PermissionBitFactoryImpl.java

```diff
@@ -624,7 +630,7 @@
 	 * Parameters
 	 * 1. structure inode
 	 */
-	private final String deleteContentReferencesByStructureSQL =
+	private static final String deleteContentReferencesByStructureSQL =
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)